app/controllers/actions/DataRequiredAction.scala

```diff
@@ -30,7 +30,7 @@ class DataRequiredActionImpl @Inject() (implicit val executionContext: Execution
   override protected def refine[A](request: OptionalDataRequest[A]): Future[Either[Result, DataRequest[A]]] =
     request.userAnswers match {
       case None =>
-        Future.successful(Left(Redirect(routes.SessionExpiredController.onPageLoad())))
+        Future.successful(Left(Redirect(routes.ErrorController.technicalDifficulties())))
```

Comment:
Can we carry on redirecting to session expired here instead?

Reply:
Updated now